packages/react/src/format.js

```diff
@@ -30,7 +48,20 @@ function formatElements(
   if (before) tree.push(before)
 
   for (const [index, children, after] of getElements(parts)) {
-    const element = elements[index]
+    let element = elements[index]
+
+    if (!element || (voidElementTags[element.type] && children)) {
+      if (!element) {
+        console.error(`Can use element at index '${index}' as it is not declared in the original translation`)
+      } else {
+        console.error(`${element.type} is a void element tag therefore it must have no children`)
+      }
+
+
+      // ignore problematic element but push its children and elements after it
+      element = createElement(Fragment)
```

Comment:
I tried few approaches but to make code readable, without many branches, I decided to create an empty element.